src/components/tags/Tag.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+
+import { ButtonColor } from '../../styles/button-theme'
+import { fontSize } from '../../styles/theme'
+import { SecondaryButton } from '../buttons/SecondaryButton'
+import { ArrayParam, useQueryParams, withDefault, encodeQueryParams, QueryParamConfig } from 'use-query-params';
+import { urls } from '../../domain/website';
+import { navigate } from '@reach/router';
+import { stringify } from 'query-string';
+
+interface ITag {
+  label?: string;
+  active?: boolean
+}
+
+interface IQuery {
+  tags: string[],
+}
+
+const Tag: React.FC<ITag> = ({ label, active }) => {
+  const [query] = useQueryParams<IQuery>({ tags: withDefault(ArrayParam, []) });
+
+  const onClick = () => {
+    if (label) {
+      let tags = query.tags.slice()
+      const isTag = tags.find((tag: string) => tag === label);
+      if (!isTag) {
+        tags.push(label)
+      } else {
+        tags = tags.filter((tag: string) => tag !== label)
+      }
+      const encodedQuery = encodeQueryParams({ tags: withDefault(ArrayParam, []) }, { tags });
+      navigate(`${urls.pola.news}?${stringify(encodedQuery)}`)
+    }
+  }
+
+  return (
+    <SecondaryButton
```

Comment:
Moim zdaniem nie używać do tagów komponentu przycisku. Lepiej coś nowego Jak TagButton... Bo to jakkolwiek jest przycisk, nawet wizualnie identyczne z przyciskiem, to jego ewolucja w przyszłości może przebiegać zupełnie inaczej niż przycisk. A wówczas zmiany w działaniu / wyglądzie taga mogą popsuć przyciski na stronie. I vice versa.